include/mbedtls/config.h

```diff
@@ -88,6 +88,16 @@
  */
 //#define MBEDTLS_NO_UDBL_DIVISION
 
+/**
+ * \def MBEDTLS_SIZE_OPTIMIZED_MULADDC
```

Comment:
Apart from what @gilles-peskine-arm has already stated, please note, that if a config option is added, it must also be registered in `mbedtls/programs/ssl/query_config.c` program.

Reply:
And note that the way to do this is to run `scripts/generate_query_config.pl`, not to edit the file manually.

But [as I noted in my review](https://github.com/ARMmbed/mbedtls/pull/1718#discussion_r305483156) I'm not sure we should add a configuration option. Every build-time option is an additional maintenance burden. We should add an option only if there is a significant tradeoff between code size and performance.